data/libs/Equipment.lua

```diff
@@ -755,6 +755,24 @@ misc.hull_autorepair = EquipType.New({
 	l10n_key="HULL_AUTOREPAIR", slots="hull_autorepair", price=16000,
 	capabilities={mass=40, hull_autorepair=1}, purchasable=true, tech_level="MILITARY"
 })
+misc.thrusters_basic = EquipType.New({
+	l10n_key="THRUSTERS_BASIC", slots="thruster", price=30,
+	tech_level=1,
+	capabilities={mass=0, thruster_power=1}, purchasable=true,
+	icon_name="thrusters_basic"
```

Comment:
If instead of `thruster_power=1` have the value of it be `=1.1` would that not save you of using the array `power_bands` later on? Oh, well, just a thought. It depends if it is a bug or feature to go beyond the 1.1, 1.2, 1.3 improvements that are now hard coded into c++ side.